app/client/components/payment/update/card/flexCardElement.tsx

```diff
@@ -33,7 +35,7 @@ export const FlexCardElement = (props: FlexCardElementProps) => (
             style: baseStyle,
             placeholder: "Card Number"
           }}
-          onReady={props.markElementReady("CardNumberElement")}
+          onReady={() => props.markCardNumberReady(true)}
```

Comment:
just a though but we could change these new bits of state to hold the actual element given by the `onReady` function (with type `undefined | StripeElementBase`) and appropriate names (e.g. `cardNumberReady` -> `cardNumberElement`) which would allow us to do something like
```suggestion
          onReady={props.setCardNumberElement}
```